### PR TITLE
8296248: Update CLDR to Version 43.0

### PR DESCRIPTION
**Pull request by naotoj**
Upgrading the CLDR to [version 43](https://cldr.unicode.org/index/downloads/cldr-43). This semi-annual release is their `limited-submission` release so I would not expect regressions caused by formatting changes as we had in JDK20/CLDRv42 (https://inside.java/2023/03/28/quality-heads-up/)

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8296248](https://bugs.openjdk.org/browse/JDK-8296248): Update CLDR to Version 43.0

### Reviewers
 * @srl295 (no known openjdk.org user name / role) ⚠️ Review applies to [d61ccf0f](https://git.openjdk.org/jdk/pull/13469/files/d61ccf0f436fbff763143aed186606851e63a437)
 * [Joe Wang](https://openjdk.org/census#joehw) (@JoeWang-Java - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13469/head:pull/13469` \
`$ git checkout pull/13469`

Update a local copy of the PR: \
`$ git checkout pull/13469` \
`$ git pull https://git.openjdk.org/jdk.git pull/13469/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13469`

View PR using the GUI difftool: \
`$ git pr show -t 13469`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13469.diff">https://git.openjdk.org/jdk/pull/13469.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13469#issuecomment-1507587249)

### CODE REVIEW

File: make/data/cldr/common/properties/coverageLevels.txt [hunk 1]
```diff
@@ -1,141 +1,143 @@
...
+sc ;	moderate ;	Sardinian
+sd ;	modern ;	Sindhi
+sd_Deva ;	basic ;	Sindhi (Devanagari)
+si ;	modern ;	Sinhala
+sk ;	modern ;	Slovak
+sl ;	modern ;	Slovenian
+so ;	modern ;	Somali
+sq ;	modern ;	Albanian
+sr ;	modern ;	Serbian
+sr_Latn ;	modern ;	Serbian (Latin)
```
Comment (srl295) [inline make/data/cldr/common/properties/coverageLevels.txt:115]: @naotoj BTW this was fixed

File: src/java.base/share/legal/cldr.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## Unicode Common Local Data Repository (CLDR) v42
+## Unicode Common Local Data Repository (CLDR) v43
```
Comment (srl295) [inline src/java.base/share/legal/cldr.md:1]: BTW the license is now just named `LICENSE` in the repo starting with v44

**Review by srl295 — APPROVED** [review]

File: make/data/cldr/common/main/ken.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		<version number="$Revision$"/>
+		<language type="ken"/>
+	</identity>
+	<localeDisplayNames>
+		<languages>
+			<language type="ken" draft="contributed">Kɛnyaŋ</language>
+		</languages>
+	</localeDisplayNames>
+	<characters>
+		<exemplarCharacters>[a á à ǎ b c d e é è ě ɛ {ɛ\u0301} {ɛ\u0300} {ɛ\u030C} f g {gb} {gh} h i ɨ {ɨ\u0301} {ɨ\u0300} {ɨ\u030C} j k {kp} m n {ny} ŋ o ó ò ǒ ɔ {ɔ\u0301} {ɔ\u0300} {ɔ\u030C} p r s t u ú ù ǔ ʉ {ʉ\u0301} {ʉ\u0300} {ʉ\u030C} w y]</exemplarCharacters>
```
[the remark this answers is not in the document]
Comment (naotoj) [reply make/data/cldr/common/main/ken.xml:19]: JDK keeps CLDR sources intact. On building JDK, .xml files are transformed into ResourceBundles and those not at the basic level are filtered out.

File: make/data/cldr/common/properties/coverageLevels.txt (hunk 1 above)
Comment (naotoj) [reply make/data/cldr/common/properties/coverageLevels.txt:115]: Yes. The patch before was removed with this commit: https://github.com/openjdk/jdk/pull/13469/commits/0e3456c8924960dc0bc488b08b9763bd08557a68

**Pull request closed without merging**